src/Microsoft.PowerShell.Commands.Management/commands/management/Clipboard.cs

```diff
@@ -28,7 +36,15 @@ private static string StartProcess(
             startInfo.RedirectStandardOutput = true;
             startInfo.RedirectStandardError = true;
             startInfo.FileName = tool;
-            startInfo.Arguments = args;
+
+            if (args != null)
+            {
+                foreach (Object arg in args)
+                {
+                    startInfo.ArgumentList.Add(Convert.ToString(arg));
```

Comment:
Args are now passed with an array, is this a good way to put it into `startInfo.ArgumentList`?